eo-runtime/src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOupper_case.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package EOorg.EOeolang.EOtxt;
+
+import org.eolang.AtComposite;
+import org.eolang.Data;
+import org.eolang.Param;
+import org.eolang.PhDefault;
+import org.eolang.Phi;
+import org.eolang.XmirObject;
+
+/**
+ * UPPER_CASE.
+ *
+ * @checkstyle TypeNameCheck (5 lines)
+ * @since 1.0
+ */
+@XmirObject(oname = "text.upper-case")
+public class EOtext$EOupper_case extends PhDefault {
+
+    /**
+     * Ctor.
+     *
+     * @param sigma Sigma
+     */
+    public EOtext$EOupper_case(final Phi sigma) {
+        super(sigma);
+        this.add(
+                "φ",
+                new AtComposite(
```

Comment:
@EugeneDar what happened with the indentation? why not four spaces? we use four spaces always.

Reply:
@yegor256 I will fix it.